src/scripts/_chunk/_chunk.gml

```diff
@@ -0,0 +1,30 @@
+/// @func _chunk(array, size)
+/// @desc Creates a two-dimensional array of elements split into groups of given length.
+/// @param {Array} array The array to split
+/// @param {Integer} size The size of each chunk
+/// @returns {Array} The two-dimensional array of chunks
+/*
+@example
+var arr = _arrayOf(0, 1, 2, 3);
+_chunk(arr, 2);
+// => [[0, 1], [2, 3]];
+
+var arr = _arrayOf(0, 1, 2, 3);
+_chunk(arr, 3);
+// => [[0, 1, 2], [3]];
+*/
+
+if (argument_count != 2) return false;
+if (argument1 == 0) return _arrayOf();
+
+var i;
+var _len;
+var __chunk;

```

Comment:
the leading and underscore and double underscore naming convention is new to this repo. Is that to avoid variable collisions? Scoping in scripts will fix that problem - if you `var` to initialize a variable in a script, it overwrites any instance variable in the scope of the script without affecting the instance.